openid-4-verifiable-presentations-1_0.md

```diff
@@ -698,6 +698,16 @@ this object are defined per Credential Format. Examples of those are in (#sd_jwt
 no specific constraints are placed on the metadata or validity of the requested
 Credential.
 
+`trusted_authorities`:
+: OPTIONAL. A non-empty array of objects as defined in (#dcql_trusted_authorities) that
```

Comment:
In allegato this pagamento agraph, issuer -> Issuer since It Is intended to be credential issuers

Reply:
Fixed in latest commit